index.html

```diff
@@ -34,6 +35,185 @@ <h1 class="text_title item-1">Vanessa Bonito</h1>
+                        <ul class="ul-style">
+                            <li class="li-style">HTML/CSS</li>
+                            <li class="li-style">Ruby on rails</li>
+                            <li class="li-style">JavaScript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="flex-button-work">
+                    <div class="button-work">
+                        See this project
+                    </div>
+                    <div class="arrow_icon flex-button-icon"></div>
+                </div>
+            </div>
+        </section>
+        <section class="grid-about">
+            <div class="div-title">
+                <h1 class="text_titleH2">About</h1>
+            </div>
+            <div class="card-about">
+                <p class="text_style">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                    et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                    aliquip ea eiusmod
+                </p>
+                <div class="flex-button-about dark_button">
+                    <div class=".button-about">
+                        Get my CV
+                    </div>
+                    <div class="download_icon flex-button-icon-about"></div>
+                </div>
+            </div>
+            <div class="card-about-abilities card-about-backgroundimg card-about-bgimg-source1">
+                <h2 class="text-tytleAbout title-about">Languages</h2>
+                <ul class="ul-about">
+                    <li class="li-about">JavaScript (ES5/ES6)</li>
+                    <li class="li-about">Ruby</li>
+                    <li class="li-about">HTML</li>
+                    <li class="li-about">CSS</li>
+                </ul>
+            </div>
+            <div class="card-about-abilities card-about-backgroundimg card-about-bgimg-source2">
+                <h2 class="text-tytleAbout title-about">Frameworks</h2>
+                <ul class="ul-about">
+                    <li class="li-about">Bootstrap</li>
+                    <li class="li-about">Ruby on rails</li>
+                    <li class="li-about">RSpec</li>
+                    <li class="li-about">Capibara</li>
+                    <li class="li-about">Selenium</li>
+                </ul>
+            </div>
+            <div class="card-about-abilities card-about-backgroundimg card-about-bgimg-source3">
+                <h2 class="text-tytleAbout title-about">Skills</h2>
+                <ul class="ul-about">
+                    <li class="li-about">CodeKit</li>
+                    <li class="li-about">Codepen</li>
+                    <li class="li-about">GitHub</li>
+                    <li class="li-about">GitLab</li>
+                    <li class="li-about">Terminal</li>
+                </ul>
+            </div>
```

Comment:
Let's put all the technology categories into a `<ul>` and make each technology category as a `<li>`. I am sure that would make more sense. 😊